<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startConnection">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Connecting</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>671</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="stun_domain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>STUNServer domain:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stun_domainEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.4.142</string>
    </property>
   </widget>
   <widget class="QLabel" name="stun_port">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>STUNServer port:</string>
    </property>
   </widget>
   <widget class="QLabel" name="signal_ip">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SignalServer ip:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="signal_ipEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.4.142</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>540</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stun_portEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>3478</string>
    </property>
   </widget>
   <widget class="QLabel" name="signal_port">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SignalServer port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="signal_portEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>8888</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>430</y>
      <width>671</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="send">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>590</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Display in Hex</string>
    </property>
   </widget>
   <zorder>startConnection</zorder>
   <zorder>stun_domain</zorder>
   <zorder>stun_domainEdit</zorder>
   <zorder>stun_port</zorder>
   <zorder>signal_ip</zorder>
   <zorder>signal_ipEdit</zorder>
   <zorder>clear</zorder>
   <zorder>textEdit</zorder>
   <zorder>stun_portEdit</zorder>
   <zorder>signal_port</zorder>
   <zorder>signal_portEdit</zorder>
   <zorder>textEdit_2</zorder>
   <zorder>send</zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>968</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
